server/config/toml.go

```diff
@@ -229,7 +232,9 @@ func ParseConfig(v *viper.Viper) (*Config, error) {
 func SetConfigTemplate(customTemplate string) {
 	var err error
 
-	tmpl := template.New("appConfigFileTemplate")
+	tmpl := template.New("appConfigFileTemplate").Funcs(template.FuncMap{
+		"StringsJoin": strings.Join,
+	})
```

Comment:
i reverted this line and the tests still passed, do we need this change? (im not to well versed in this side of the code so i could be very wrong)

Reply:
Good catch. I was copying stuff from the Tendermint config `toml.go` and thought that was needed, but it's not. I've removed it and also added a unit test on the `SetConfigTemplate` function to hopefully catch something being changed there and not `init()` or vice versa.

For reference, that `"StringsJoin"` is only used in the Tendermint config in one spot: `rpc_servers = "{{ StringsJoin .StateSync.RPCServers "," }}"`. The `StateSync.RPCServers` field is a `[]string` but looks like a single string in the config file. So that's not the correct thing to use for `index-events` where we still want it to be a list of strings in the config file.

Reply:
ah i see! thank you for the explanation 😄